src/components/UpdateArticle.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import categoryList from "../modules/category";
+import { Container, Grid, Form, Dropdown, Button } from "semantic-ui-react";
+import Preview from "./Preview";
+
+const UpdateArticle = (props) => {
+  const categories = categoryList();
+  return (
+    <Grid columns={2}>
+      <Grid.Column id="left">
+        <Container>
+          <Preview selectedArticle={props.selectedArticle} />
+        </Container>
+      </Grid.Column>
+      <Grid.Column>
+        <Form onSubmit={props.onSubmitHandler}>
```

Comment:
Do you really need the form if you are not using e.target in the receiver function? I think you should either use e.target for selection ( and do the thing I suggested with the radio buttons ), or remove the form and move the handler to the button

Reply:
e.target is used to toggle on and off the radio buttons, it is tracking the change
If one them is selected, selecting the other one de-selects the first one